dora/src/semck/prelude.rs

```diff
@@ -439,7 +456,7 @@ fn internal_method<'ast>(vm: &mut VM<'ast>, clsid: ClassId, name: &str, kind: Fc
         let mtd = vm.fcts.idx(mid);
         let mut mtd = mtd.write();
 
-        if mtd.name == name && mtd.internal {
+        if mtd.name == name {
             mtd.kind = kind;
             mtd.internal_resolved = true;
```

Comment:
Should we set `mtd.internal = true` here too?

Reply:
Wouldn't it be cleaner to run `typeck::check` before we are adding the internal methods?

Reply:
I am still not convinced that replacing `kind` is the way to go - but since I can't name an example where we need it in the runtime, we can do this.